ag/ag.proto

```diff
@@ -297,6 +298,14 @@ message Organizations {
     repeated Organization organizations = 1;
 }
 
+// TODO(vera): rename this message to smth more descriptive
+// UpdateTokenRecord stores ID of a user whose token requires an update
+// next time the user logs in
+message UpdateTokenRecord {
```

Comment:
Could we instead add a field to the `User` type to indicate whether or not a user's JWT token is valid?

We have database methods that can return a single user or a slice of users:
- `GetUser(userID uint64) *pb.User`
- `GetUsers(userIDs ...uint64) []*pb.User`

Reply:
Agree that this is a better solution. Updated.